server/sonar-web/src/main/js/apps/issues/components/BulkChangeModal.js

```diff
@@ -109,44 +109,40 @@ export default class BulkChangeModal extends React.PureComponent {
     this.mounted = false;
   }
 
+  loadIssues = () => this.props.fetchIssues({ additionalFields: 'actions,transitions', ps: 250 });
+
+  getDefaultAssignee = () => {
+    const { currentUser } = this.props;
+    const { issues } = this.state;
+    const options = [];
+
+    if (currentUser.isLoggedIn) {
+      const canBeAssignedToMe =
+        issues.filter(issue => issue.assignee !== currentUser.login).length > 0;
+      if (canBeAssignedToMe) {
+        options.push({
+          avatar: currentUser.avatar,
+          label: currentUser.name,
+          value: currentUser.login
+        });
+      }
+    }
+
+    const canBeUnassigned = issues.filter(issue => issue.assignee).length > 0;
+    if (canBeUnassigned) {
+      options.push({ label: translate('unassigned'), value: '' });
+    }
+
+    return options;
+  };
+
   handleCloseClick = (e /*: Event & { target: HTMLElement } */) => {
     e.preventDefault();
     e.target.blur();
     this.props.onClose();
   };
 
-  loadIssues = () => {
-    return this.props.fetchIssues({ additionalFields: 'actions,transitions', ps: 250 });
-  };
-
-  handleAssigneeSearch = (query /*: string */) => {
-    if (query.length > 1) {
-      return searchAssignees(query, this.state.organization);
-    } else {
-      const { currentUser } = this.props;
-      const { issues } = this.state;
-      const options = [];
-
-      if (currentUser.isLoggedIn) {
-        const canBeAssignedToMe =
-          issues.filter(issue => issue.assignee !== currentUser.login).length > 0;
-        if (canBeAssignedToMe) {
-          options.push({
-            email: currentUser.email,
-            label: currentUser.name,
-            value: currentUser.login
-          });
-        }
-      }
-
-      const canBeUnassigned = issues.filter(issue => issue.assignee).length > 0;
-      if (canBeUnassigned) {
-        options.push({ label: translate('unassigned'), value: '' });
-      }
-
-      return Promise.resolve(options);
-    }
-  };
+  handleAssigneeSearch = (query /*: string */) => searchAssignees(query, this.state.organization);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove parentheses around the parameter of this arrow function. ([javascript:S3524](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarqube%3Asonarqube&branch=feature%2Fgreg%2Fhardening%2FSONAR-10298&issues=AWFHvAyvUvOPqYRWG8zp&open=AWFHvAyvUvOPqYRWG8zp)

[ISSUE_KEY]: AWFHvAyvUvOPqYRWG8zp

Reply:
This issue has been resolved by a user in SonarQube

[ISSUE_STATUS]: RESOLVED